src/components/calcite-popover/calcite-popover.tsx

```diff
@@ -139,7 +139,7 @@ export class CalcitePopover {
   }
 
   /**
-   * Reference HTMLElement used to position this component according to the placement property.
+   * Reference HTMLElement used to position this component according to the placement property. As a convenience, a string id of the reference element can be used. However, setting this property to use an HTMLElement is preferred so that the component does not need to query the DOM for the referenceElement.
```

Comment:
`id` => `ID`

> However, setting this property to use an HTMLElement is preferred so that the component does not need to query the DOM for the referenceElement.

IMO, we can drop this. Seems like an implementation detail. I'll leave it up to you.